### DISCUSSION TITLE
Bug in FastAPI Documentation https://fastapi.tiangolo.com/reference/fastapi/

### DISCUSSION THREAD

**Discussion by imthematrix** [fastapi/fastapi#11257] [Questions]
### Description

* Open [https://fastapi.tiangolo.com/reference/fastapi/](https://fastapi.tiangolo.com/reference/fastapi/)
* Under [https://fastapi.tiangolo.com/reference/fastapi/#fastapi.FastAPI](https://fastapi.tiangolo.com/reference/fastapi/#fastapi.FastAPI) there's a code block that isn't formatted right:

    `FastAPI(*, <span data-autorefs-optional="fastapi.FastAPI(debug)">debug</span>=False, <span data-autorefs-optional="fastapi.FastAPI(routes)">routes</span>=None, <span data-autorefs-optional="fastapi.FastAPI(title)">title</span>='FastAPI', ................)`

* The code contains many html elements like `<span ...>...</span>` that shouldn't be visible. It's supposed to contain python code.

### FastAPI Version

0.110.0

**Comment by YuriiMotov** [comment]
Fixed
